src/Benchmarks/FortunesDapperMiddleware.cs

```diff
@@ -46,14 +46,13 @@ public FortunesDapperMiddleware(RequestDelegate next, string connectionString, D
 
         private static async Task<IEnumerable<Fortune>> LoadRows(string connectionString, DbProviderFactory dbProviderFactory)
         {
-            var result = new List<Fortune>();
+            List<Fortune> result;
 
             using (var db = dbProviderFactory.CreateConnection())
             {
                 db.ConnectionString = connectionString;
-                await db.OpenAsync();
-
-                result.AddRange(await db.QueryAsync<Fortune>("SELECT [Id], [Message] FROM [Fortune]"));
+                // note: don't need to open connection if only doing one thing; let dapper do it
+                result = (await db.QueryAsync<Fortune>("SELECT [Id], [Message] FROM [Fortune]")).AsList();
```

Comment:
Thanks @benaadams ; I have reviewed the requirements; dapper does nothing that would violate Test 4 rule 5, if that  is the concern.